frame/composable-traits/src/dex.rs

```diff
@@ -181,7 +181,7 @@ pub type PoolId = u32;
 
 /// Pool type
 #[derive(Encode, Decode, TypeInfo, Clone, Default, PartialEq, Eq, Debug)]
-pub struct PoolInfo<AccountId> {
+pub struct StableSwapPoolInfo<AccountId> {
 	/// Owner of pool
 	pub owner: AccountId,
```

Comment:
is it viable to unify owner and admin_fee under one struct? so it can be reused?

Reply:
I don't see a huge benefit. if required it can be done later.